bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -5,6 +5,6 @@
 
 	<name>OpenWeatherMap Binding</name>
 	<description>OpenWeatherMap - Current weather and forecasts in your city.</description>
-	<author>Christoph Weitkamp</author>
+	<author>Christoph Weitkamp, Wolfgang Klimt</author>
```

Comment:
One final comment from my side. The `author`-tag now is deprecated. Please remove it. Thanks for your efforts.

Reply:
Done.